definitions/v3/fuzer.yml

```diff
@@ -0,0 +1,154 @@
+    - {id: 55, cat: Console/XBox 360, desc: "Console/XBox"}
+    - {id: 55, cat: Console/XBox 360 DLC, desc: "Console/XBox"}
+    - {id: 12, cat: Console/PS3, desc: "Console/PS"}
+    - {id: 12, cat: Console/PS4, desc: "Console/PS"}
+    - {id: 56, cat: Console/Wii, desc: "Console/Wii"}
+    - {id: 57, cat: Console/3DS, desc: "Console/Mobile"}
+    - {id: 57, cat: Console/WiiU, desc: "Console/Mobile"}
+    - {id: 57, cat: Console/PS Vita, desc: "Console/Mobile"}
+    - {id: 57, cat: Console/NDS, desc: "Console/Mobile"}
+    - {id: 57, cat: Console/PSP, desc: "Console/Mobile"}
+
+    # Music
+    - {id: 75, cat: Audio, desc: "Music"}
+    - {id: 14, cat: Audio, desc: "World Music"}
+    - {id: 66, cat: Audio, desc: "Israeli Music"}
+    - {id: 68, cat: Audio, desc: "Soundtracks"}
+    - {id: 67, cat: Audio/Lossless, desc: "Lossless Music"}
+
+    # Books
+    - {id: 69, cat: Books, desc: "Books"}
+
+    # Other
+    - {id: 17, cat: Other, desc: "Other"}
+
+  modes:
+    search: [q]
+    tv-search: [q, imdbid]
+    movie-search: [q, imdbid]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: cookie
+    type: text
+    label: Cookie
+  - name: info
+    type: info
+    label: How to get the Cookie
+    default: "<ol><li>Login to this tracker with your browser<li>Open the <b>DevTools</b> panel by pressing <b>F12</b><li>Select the <b>Network</b> tab<li>Click on the <b>Doc</b> button (Chrome Browser) or <b>HTML</b> button (FireFox)<li>Refresh the page by pressing <b>F5</b><li>Click on the first row entry<li>Select the <b>Headers</b> tab on the Right panel<li>Find <b>'cookie:'</b> in the <b>Request Headers</b> section<li><b>Select</b> and <b>Copy</b> the whole cookie string <i>(everything after 'cookie: ')</i> and <b>Paste</b> here.</ol>"
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: dead
+    type: checkbox
+    label: Include dead torrents
+    default: false
+
+login:
+  method: cookie
+  inputs:
+    cookie: "{{ .Config.cookie }}"
+  test:
+    path: /
+    selector: a[href="/login.php"]
+
+search:
+  paths:
+    - path: "browse.php"
+  inputs:
+    query: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}"
+    order: dateadde
+    sort: desc
+    matchquery: any
+    freeleech: "{{ if .Config.freeleech }}1{{ else }}0{{ end }}"
+    dead: "{{ if .Config.dead }}on{{ else }}off{{ end }}"
+    $raw: "{{ range .Categories }}c[]={{.}}&{{end}}"
+  rows:
+    selector: .box_torrent
+
+  fields:
+    title:
+      selector: .main_title
+    id:
+      selector: a[href^="/showthread.php?t="]
+      attribute: href
+      filters:
+        - name: split
+          args: ["=", 1]
+    details:
+      selector: a[href^="/showthread.php?t="]
+      attribute: href
+    download:
+      selector: a[href^="/attachment.php?attachmentid="]
+      attribute: href
+    size:
+      selector: td:nth-child(4)
+    grabs:
+      selector: td:nth-child(5)
```

Comment:
when there are 1000 or more grabs, i'm facing the following error
`Error while parsing field=grabs, selector=td:nth-child(5), value=1,683: Value was either too large or too small for an Int32.`
how should i solve it?

Reply:
@Qstick do we need to change cardigann to use Long instead?

Reply:
It needs to not have a comma

Reply:
Yes the parser doesn't like ints with commas, do a replace filter on that prop in the yml to replace `","` with `""`

Reply:
thanks.

replaced commas in relevant fields